diff --git a/dev/Enemies Notd2.SC2Mod/ComponentList.SC2Components b/dev/Enemies Notd2.SC2Mod/ComponentList.SC2Components
index 9408870..f765f5c 100644
--- a/dev/Enemies Notd2.SC2Mod/ComponentList.SC2Components	
+++ b/dev/Enemies Notd2.SC2Mod/ComponentList.SC2Components	
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="us-ascii"?>
 <Components>
-    <Optimized/>
     <DataComponent Type="gada">GameData</DataComponent>
     <DataComponent Type="text" Locale="enUS">GameText</DataComponent>
     <DataComponent Type="info">DocumentInfo</DataComponent>
diff --git a/dev/NOTD 2 Sounds.SC2Mod/Base.SC2Data/GameData/SoundData.xml b/dev/NOTD 2 Sounds.SC2Mod/Base.SC2Data/GameData/SoundData.xml
index 4c72401..e31656a 100644
--- a/dev/NOTD 2 Sounds.SC2Mod/Base.SC2Data/GameData/SoundData.xml	
+++ b/dev/NOTD 2 Sounds.SC2Mod/Base.SC2Data/GameData/SoundData.xml	
@@ -408,19 +408,19 @@
         <AssetArray>
             <File value="widow_maker_shot_01.wav"/>
             <Pitch value="-1.000000,1.000000"/>
-            <Volume value="-8.000000,-8.000000"/>
+            <Volume value="-10.000000,-9.000000"/>
             <FacialGroup value=""/>
         </AssetArray>
         <AssetArray>
             <File value="widow_maker_shot_02.wav"/>
             <Pitch value="-1.000000,1.000000"/>
-            <Volume value="-8.000000,-8.000000"/>
+            <Volume value="-10.000000,-9.000000"/>
             <FacialGroup value=""/>
         </AssetArray>
         <AssetArray>
             <File value="widow_maker_shot_03.wav"/>
             <Pitch value="-1.000000,1.000000"/>
-            <Volume value="-9.000000,-8.000000"/>
+            <Volume value="-10.000000,-9.000000"/>
             <FacialGroup value=""/>
         </AssetArray>
         <Flags index="Purgable" value="0"/>
diff --git a/dev/NOTD 2 Sounds.SC2Mod/ComponentList.SC2Components b/dev/NOTD 2 Sounds.SC2Mod/ComponentList.SC2Components
index e209a2d..d74aef6 100644
--- a/dev/NOTD 2 Sounds.SC2Mod/ComponentList.SC2Components	
+++ b/dev/NOTD 2 Sounds.SC2Mod/ComponentList.SC2Components	
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="us-ascii"?>
 <Components>
-    <Optimized/>
     <DataComponent Type="gada">GameData</DataComponent>
     <DataComponent Type="text" Locale="enUS">GameText</DataComponent>
     <DataComponent Type="info">DocumentInfo</DataComponent>
diff --git a/dev/NOTD 2 Sounds.SC2Mod/DocumentHeader b/dev/NOTD 2 Sounds.SC2Mod/DocumentHeader
index 4f1d414..982d936 100644
Binary files a/dev/NOTD 2 Sounds.SC2Mod/DocumentHeader and b/dev/NOTD 2 Sounds.SC2Mod/DocumentHeader differ
diff --git a/dev/NOTD 2 Sounds.SC2Mod/PreloadAssetDB.xml b/dev/NOTD 2 Sounds.SC2Mod/PreloadAssetDB.xml
index 608f4db..80797c8 100644
--- a/dev/NOTD 2 Sounds.SC2Mod/PreloadAssetDB.xml	
+++ b/dev/NOTD 2 Sounds.SC2Mod/PreloadAssetDB.xml	
@@ -5,6 +5,29 @@ path=Assets\Sounds\.wav
 1=Space_DoorExplosion_2
 2=Space_DoorExplosion_3
 Actor=TerrainMgr
+id=AdvancedTargeting2
+0=Assault Advanced Targeting.wav
+Actor=TerrainMgr
+id=Execute
+path=Assets\Sounds\Protoss\DarkTemplar\.wav
+0=DarkTemplar_AttackLaunch0
+1=DarkTemplar_AttackLaunch1
+2=DarkTemplar_AttackLaunch2
+3=DarkTemplar_AttackLaunch4
+Actor=TerrainMgr
+id=Refresher
+path=Assets\Sounds\Protoss\Scout\.wav
+0=Scout_AirAttackLaunch0
+1=Scout_AirAttackLaunch1
+2=Scout_AirAttackLaunch2
+3=Scout_AirAttackLaunch3
+Actor=TerrainMgr
+id=FlareGun
+0=Grenade_Launcher_Fire_2.wav
+Actor=TerrainMgr
+id=Escape
+0=Assets\Sounds\Protoss\Mothership\Mothership_WormholeTransitStart0.wav
+Actor=TerrainMgr
 id=Provoke
 0=Demo_##id##.wav
 Actor=TerrainMgr
@@ -259,29 +282,6 @@ id=UGC_Has_Arrived
 Actor=TerrainMgr
 id=Uncertainty
 Actor=TerrainMgr
-id=AdvancedTargeting2
-0=Assault Advanced Targeting.wav
-Actor=TerrainMgr
-id=FlareGun
-0=Grenade_Launcher_Fire_2.wav
-Actor=TerrainMgr
-id=Escape
-0=Assets\Sounds\Protoss\Mothership\Mothership_WormholeTransitStart0.wav
-Actor=TerrainMgr
-id=Execute
-path=Assets\Sounds\Protoss\DarkTemplar\.wav
-0=DarkTemplar_AttackLaunch0
-1=DarkTemplar_AttackLaunch1
-2=DarkTemplar_AttackLaunch2
-3=DarkTemplar_AttackLaunch4
-Actor=TerrainMgr
-id=Refresher
-path=Assets\Sounds\Protoss\Scout\.wav
-0=Scout_AirAttackLaunch0
-1=Scout_AirAttackLaunch1
-2=Scout_AirAttackLaunch2
-3=Scout_AirAttackLaunch3
-Actor=TerrainMgr
 
 [Soundtrack]
 id=Chapter0Ambience
